### DISCUSSION TITLE
How to Implement the Wishlist Plugin in MedusaJS?

### DISCUSSION THREAD

**Discussion by bernabedev** [medusajs/medusa#7329] [Q&A]
Hi everyone,

I am trying to implement the wishlist plugin in MedusaJS, but I have encountered an issue. According to the documentation, I should be able to call routes like `/store/customers/<CUSTOMER_ID>/wishlist`, but it seems that the endpoint does not exist or is not working correctly. Response like this `Cannot POST /store/customers/cus_01HXPQBBNT01A632/wishlist`.

## Points I Have Verified:
1. **Plugin Installation Confirmation**:
    - I have installed and configured the wishlist plugin following the instructions from the official MedusaJS documentation.

2. **Documentation Review**:
    - I have reviewed the documentation and I am sure that I am calling the routes correctly.

3. **Route Configuration**:
    - I have checked the routes in my configuration file, but there does not seem to be any obvious errors.

4. **Logs and Errors**:
    - I have checked the server logs, but I do not see any error messages related to the wishlist plugin.

5. **Version Verification**:
    - I am using the latest version of MedusaJS, which should be compatible with the wishlist plugin.

## Discussion Questions:
- Has anyone had a similar experience when trying to implement this plugin?
- Is there an additional step that I might be missing?
- Could it be a configuration issue that I am not seeing?
- Does anyone have any suggestions or alternative solutions while we resolve this issue?

I appreciate any help or advice you can offer. Thanks!

**Comment by buffcode** [comment]
We discovered that our local VMs and QA deployment work correctly but on production we get 404 / `Cannot POST /store/customers/cus_01J..../wishlist`. We are still investigating why.

Did you find out what was wrong on your side? It seems to be a deployment or configuration issue, as the plugin is working correctly on some systems.

Thanks in advance!

**Reply by amaben2020** [reply]
Hey @buffcode , has this been resolved? I'm getting a 404 Cannot POST /store/customers/cus_01JJ7WT2S2BTMW41SAEF2ANB6Q/wishlist

**Comment by radoslaw-sz** [comment]
hi all - if you are interested, there is a plugin released by my company under MIT licence - https://github.com/RSC-Labs/medusa-wishlist

**Comment by byteInventor** [comment]
Hi, any update on this issue? did anyone of you guys fixed it?

**Comment by zamanehsani** [comment]
I got the same error. it seems that the plugin does not work.
